client/src/main/scala/com.gu.contentapi.client/model/Queries.scala

```diff
@@ -11,6 +11,15 @@ sealed trait ContentApiQuery {
   override def toString = {
     s"${getClass.getSimpleName}(/$pathSegment${QueryStringParams(parameters)})"
   }
+
+  def url(location: String, parameters: Map[String, String]): String = {
```

Comment:
I don't think this method should be public. It doesn't add much value to callers and seems error-prone. The method below it (`apply`) is more useful in that it makes use of the `pathSegment`.